LeaderboardBackend/Filters/EnumTypesSchemaFilter.cs

```diff
@@ -0,0 +1,62 @@
+using System.Xml.Linq;
+using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace LeaderboardBackend.Filters;
+
+/// <summary>
+/// Reflects the contents of <c>summary</c> tags in enum fields' doc comments
+/// to the enum's schema definition in the Swagger doc. I.e. this class allows
+/// us to describe an enum's values in its schema definition in the doc.
+/// <br/>
+/// Adapted from <see href="https://www.codeproject.com/Articles/5300099/Description-of-the-Enumeration-Members-in-Swashbuc"/>.
```

Comment:
Is it okay to use his code? He didn't provide an open source license.

Reply:
I've dropped the filter